test/typings.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { createStore, applyMiddleware, MiddlewareAPI, Action } from 'redux';
+import { createStore, applyMiddleware, Action } from 'redux';
 import { Observable, asapScheduler, of } from 'rxjs';
```

Comment:
Do we still need this test/typings.ts file? I don't think we would, since the real tests are now in TypeScript.